website/docs/r/app_oauth.html.markdown

```diff
@@ -57,11 +57,15 @@ The following arguments are supported:
 
 - `app_settings_json` - (Optional) Application settings in JSON format.
 
-- `auto_key_rotation` - (Optional) Requested key rotation mode. See: https://developer.okta.com/docs/reference/api/apps/#oauth-credential-object
+- `auto_key_rotation` - (Optional) Requested key rotation mode.  If
+    `auto_key_rotation` isn't specified, the client automatically opts in for Okta's
+    key rotation. You can update this property via the API or via the administrator
+    UI.
+    See: https://developer.okta.com/docs/reference/api/apps/#oauth-credential-object
 
 - `auto_submit_toolbar` - (Optional) Display auto submit toolbar.
 
-- `client_basic_secret` - (Optional) OAuth client secret key, this can be set when token_endpoint_auth_method is client_secret_basic.
+- `client_basic_secret` - (Optional) OAuth client secret key, this can be set when `token_endpoint_auth_method` is `"client_secret_basic"`.
```

Comment:
nit: Shouldn't this be `client_secret_basic`?

Reply:
`client_basic_secret` corresponds to `client_secret` the way okta-sdk-golang maps onto the API and in turn the TF provider defines its schema https://developer.okta.com/docs/reference/api/apps/#username-template-object . Also, even if the argument was mis-named I can't go renaming it now without breaking everyone's configs when they upgrade to the version this change is in.